utils/swift_build_support/tests/test_toolchain.py

```diff
@@ -26,6 +27,18 @@ def test_clang_available_on_this_platform(self):
         self.assertTrue(os.path.split(clang.cc)[-1].startswith('clang'))
         self.assertTrue(os.path.split(clang.cxx)[-1].startswith('clang++'))
 
+    def test_found_executables_match(self):
```

Comment:
nit-pick: This doesn't test `host_clang()`, it tests `host_toolchain()`. As such, I'd recommend putting it in a separate `class HostToolchainTestCase(unittest.TestCase)`.